csrc/convert.cpp

```diff
@@ -10,12 +10,14 @@
 #endif
 
 #ifdef _WIN32
+#ifdef WITH_PYTHON
```

Comment:
Sorry, only noticed when building on WIndows that we also need to add this to avoid the Python dependency.